README.md

```diff
@@ -929,6 +929,31 @@ config.after(:suite) do
   FileUtils.rm_rf(Dir["#{Rails.root}/spec/test_files/"])
 end
 ```
+
+**Configurating Tests (Factory Girl)**
+
+FactoryGirl setup for the model that's getting the Paperclip attachment:
+
+```ruby
+FactoryGirl.define do
+  factory :asset do
+    attachment { Rack::Test::UploadedFile.new("#path_to_attachment", "attachment_file_type (e.g. image/png)") }
+  end
```

Comment:
Thanks! What do you think of a less generic example, like:

```ruby
factory :user do
  avatar { File.new("#{Rails.root}/spec/support/fixtures/image.jpg") }
end
```

(taken from https://stackoverflow.com/questions/3294824/how-do-i-use-factory-girl-to-generate-a-paperclip-attachment)